src/main.cpp

```diff
@@ -334,6 +335,28 @@ static std::optional<struct timespec> get_boottime()
   if (!get_bool_env_var("BPFTRACE_VERIFY_LLVM_IR", verify_llvm_ir))
     return false;
 
+  if (const char* stack_mode = std::getenv("BPFTRACE_STACK_MODE"))
+  {
+    if (strcmp(stack_mode, "bpftrace") == 0)
+    {
+      bpftrace.stack_mode_ = StackMode::bpftrace;
+    }
```

Comment:
I'd prefer the C++ way:
```suggestion
  if (const char* env_p = std::getenv("BPFTRACE_STACK_MODE"))
  {
    std::string stack_mode(env_p);
    if (stack_mode == "bpftrace")
    {
      bpftrace.stack_mode_ = StackMode::bpftrace;
    }
```
Also, I'd suggest using `switch` rather than `if ... else if ... `.

Reply:
Switch doesn't work on strings but I"ll update the ==